config.toml

```
@@ -114,7 +111,12 @@ type  = "sans_serif"
 # Flux hero element
 [params.flux]
 # title    = "Alternative title"
-subtitle = "Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit."
+subtitle = """
+Flux 2 will soon be generally available. It is the start of Flux as a GitOps family of projects.
+Open and extensible continuous and progress delivery solutions for Kubernetes. Powered by the GitOps Toolkit.
```

```suggestion
Open, extensible continuous and progress delivery solutions for Kubernetes. Powered by the GitOps Toolkit.
```

Shaves off one "and", which reads a bit awkward.